tests/t5/test_modeling_tf_t5.py

```diff
@@ -346,6 +346,11 @@ def test_resize_embeddings(self):
         self.assertEqual(model.get_input_embeddings().weight.shape[0], len(tokenizer))
         self.assertNotEqual(model.get_input_embeddings().weight.shape[0], original_vocab_size)
 
+    # This test is run in `TFT5EncoderOnlyModelTest`, where the main layer has the same inputs as the model
+    @unittest.skip(reason="The inputs of the Main Layer are different.")
+    def test_keras_save_load(self):
+        pass
```

Comment:
Related to the `dummy_inputs` comment above. This test uses the [TFT5MainLayer](https://github.com/huggingface/transformers/blob/main/src/transformers/models/t5/modeling_tf_t5.py#L614) class, which has the same inputs as `TFT5EncoderModel`. All classes in this `Tester` use the other input format.

This test still happens below, in the `Tester` for `TFT5EncoderModel`.